src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/X86/Avx512F.cs

```diff
@@ -2125,6 +2237,11 @@ public new abstract class X64 : Avx2.X64
         /// </summary>
         public static Vector512<float> ConvertToVector512Single(Vector512<int> value) => ConvertToVector512Single(value);
         /// <summary>
+        /// __m512 _mm512_cvt_roundepi32_ps (__m512i a, int rounding)
+        ///   VCVTDQ2PS zmm1, zmm2 {er}
+        /// </summary>
+        public static Vector512<float> ConvertToVector512Single(Vector512<int> value, [ConstantExpected(Max = FloatRoundingMode.ToZero)] FloatRoundingMode mode) => ConvertToVector512Single(value, mode);
```

Comment:
Looks like we're missing the unsigned equivalent: `ConvertToVector512Single` (`VCVTUDQ2PS`)